contracts/global/LagrangeDistributor.sol

```diff
@@ -173,7 +190,7 @@ contract LagrangeDistributor is Ownable, ILPNClient, ILagrangeDistributorExcepti
         for (uint256 i = 0; i < len; ++i) {
             _campaigns[campaignId].rewards.push(rewards[i]);
             IERC20(rewards[i].rewardToken).safeTransferFrom(
-                msg.sender, address(this), rewards[i].rewardRate * (endBlock - startBlock)
+                msg.sender, address(this), rewards[i].rewardRate * (endBlock - startBlock + 1)
```

Comment:
Lagrange queries are inclusive of the range; do you want the campaign to work in the same way? e.g.

```
startBlock = 1
endBlock = 2
=> 2 blocks of rewards in the campaign
```